torch/utils/dlpack.py

```diff
@@ -3,28 +3,30 @@
 import torch
 import enum
 
-from torch._C import _from_dlpack
 from torch._C import _to_dlpack as to_dlpack
 
 __all__ = [
     "DLDeviceType",
     "from_dlpack",
-    "to_dlpack",
 ]
 
-
 class DLDeviceType(enum.IntEnum):
     # Enums as in DLPack specification (aten/src/ATen/dlpack.h)
     kDLCPU = 1,
-    kDLGPU = 2,
-    kDLCPUPinned = 3,
```

Comment:
why change the name?

Reply:
This change reflects on the `aten/src/ATen/dlpack.h` enum.